src/client/routes/Portfolio/index.tsx

```diff
@@ -141,7 +141,12 @@ export const Portfolio = () => {
     switch (action) {
       case 'invest':
         dispatch(TokensActions.setSelectedTokenAddress({ tokenAddress }));
-        dispatch(ModalsActions.openModal({ modalName: 'depositTx' }));
+        dispatch(
+          ModalsActions.openModal({
+            modalName: 'depositTx',
+            modalProps: { allowTokenSelect: false, allowVaultSelect: true },
```

Comment:
Just wondering if this is the only place we want `{ allowTokenSelect: false, allowVaultSelect: true }`

Reply:
I think so, since previously it was hard-coded to only do that on the "/wallet" path. Couldn't find anywhere else we load this modal other than the vaults page

Reply:
Nice! All good